components/cache.rst

```diff
@@ -25,112 +26,58 @@ Alternatively, you can clone the `<https://github.com/symfony/cache>`_ repositor
 
 .. include:: /components/require_autoload.rst.inc
 
-Cache (PSR-6) Versus Simple Cache (PSR-16)
-------------------------------------------
+Cache Contracts versus PSR-6
+----------------------------
 
 This component includes *two* different approaches to caching:
 
 :ref:`PSR-6 Caching <cache-component-psr6-caching>`:
-     A fully-featured cache system, which includes cache "pools", more advanced
-     cache "items", and :ref:`cache tagging for invalidation <cache-component-tags>`.
+     A generic cache system, which involves cache "pools" and cache "items".
 
-:ref:`PSR-16 Simple Caching <cache-component-psr16-caching>`:
-    A simple way to store, fetch and remove items from a cache.
-
-Both methods support the *same* cache adapters and will give you very similar performance.
+:ref:`Cache Contracts <cache-component-contracts>`:
+    A simple yet powerful way to store, fetch and remove values from a cache.
 
 .. tip::
 
-    The component also contains adapters to convert between PSR-6 and PSR-16 caches.
-    See :doc:`/components/cache/psr6_psr16_adapters`.
-
-.. _cache-component-psr16-caching:
-
-Simple Caching (PSR-16)
------------------------
-
-This part of the component is an implementation of `PSR-16`_, which means that its
-basic API is the same as defined in the standard. First, create a cache object from
-one of the built-in cache classes. For example, to create a filesystem-based cache,
-instantiate :class:`Symfony\\Component\\Cache\\Simple\\FilesystemCache`::
-
-    use Symfony\Component\Cache\Simple\FilesystemCache;
-
-    $cache = new FilesystemCache();
+    Using the Cache Contracts approach is recommended: using it requires less
+    code boilerplate and provides cache stampede protection by default.
 
-Now you can create, retrieve, update and delete items using this object::
```

Comment:
If we are going to remove these examples, we should provide a replacement with the new recommended way of doing it, right?